docs/Generics/generics.tex

```diff
@@ -161,7 +161,7 @@ \chapter{Introduction}\label{roadmap}
 
 An important part of compiler implementation is the design of domain objects to model concepts in the language being compiled. One way to think of a compiler is that it is \emph{a library for implementing the target language}. A well-designed set of domain objects facilitates the introduction of new language features that compose existing functionality in new ways.
 
-The generics implementation deals with four fundamental domain objects: \emph{generic signatures}, \emph{substitution maps}, \emph{requirement signatures}, and \emph{conformances}. As you will see, they are defined as much by their inherent structure, as their relationship with each other. Subsequent chapters will dive into all the details, but first, we're going to look at a series of worked examples to help you understand the big picture.
```

Comment:
"worked example" was intentional here

Reply:
Ah, TIL this was a phrase in English! (I'm not a native speaker)

Reply:
Reverted in https://github.com/apple/swift/pull/61604/commits/a509c7332b450c23ad2a5e0f0310742376cf914b